src/angular-vs-repeat.js

```diff
@@ -259,8 +264,23 @@
                             }
                         });
 
-
+                        var prevCollectionKeys = [];
                         $scope.$watchCollection(rhs, function(coll) {
+                            try {
+                                if(trackBy) {
+                                    // Don't refresh if tracked property of collection does not change
+                                    var currentCollectionKeys = _.map(coll, trackBy);
+                                    if(_.isEqual(currentCollectionKeys, prevCollectionKeys)) {
+                                        return;
+                                    } else {
+                                        prevCollectionKeys = currentCollectionKeys;
+                                    }
+                                }
+                            }
+                            catch (err) {
```

Comment:
@isupersid - Getting back to this now that we're in post GA.  Thanks for catching the unnecessary throw ;).  I added this while trying a few things and forgot to remove.  Cleaned up in my last commit.

Re: _.map, it's ok for it to return dupes here since I'm purely checking for ordering and size changes.  NgRepeat trackBy needs to enforce no dupes so that it can maintain DOM <-> model mapping but we don't have that requirement here.  Thanks for pointing it out though.